### PR TITLE
Automation of eventing-broker-trigger | knative-eventing

### PR DESCRIPTION
**Pull request by SparshKesari**
**Description:**

- Automation of eventing-broker-trigger | knative-eventing

**Jira Id:** 

- [ODC-6674](https://issues.redhat.com/browse/ODC-6674)

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example: 
```
export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
    BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
    BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.8-042801.devcluster.openshift.com
    export BRIDGE_KUBEADMIN_PASSWORD
    export BRIDGE_BASE_ADDRESS
    oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
    oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
    oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
    ./test-cypress.sh -p knative

```

**Screen shots**:

image

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by hemantsaini-7** [comment]
/lgtm

image

**Comment by hemantsaini-7** [comment]
/lgtm

image

File: frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-trigger.feature [hunk 1]
```diff
@@ -31,6 +18,19 @@ Feature: Knative Eventing Broker and Trigger
...
 
+        @smoke 
+        Scenario: Add Trigger to Broker: KE-04-TC02
+            Given user is having Broker "broker-test1" on the Topology page
+              And user is at Topology page
+             When user right clicks on the Broker "broker-test1" to open the context menu 
+              And user clicks on the Add Trigger
+              And user selects the auto populated name of subscription
+              And user will click on the Subscriber dropdown on the modal
+              And user selects the Subscriber
```
Comment (sanketpathak) [inline frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-trigger.feature:29]:
```suggestion
              And user selects the Subscriber "nodejs-ex-git2"
```

File: frontend/packages/knative-plugin/integration-tests/support/step-definitions/eventing/eventing-broker-trigger.ts [hunk 2]
```diff
@@ -48,3 +49,68 @@ Then(
...
+
+When('user clicks on the Delete button on the modal', () => {
+  cy.get(topologyPO.graph.deleteWorkload).click();
+});
+
+Then('trigger will get deleted', () => {
+  cy.get(topologyPO.graph.triggerLink).should('not.exist');
+});
+
+When('user selects the Subscriber from dropdown', () => {
```
Comment (sanketpathak) [inline frontend/packages/knative-plugin/integration-tests/support/step-definitions/eventing/eventing-broker-trigger.ts:109]: Where is this used in feature file

File: frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-trigger.feature [hunk 3]
```diff
@@ -31,6 +18,19 @@ Feature: Knative Eventing Broker and Trigger
              Then Subscriber has knative service dropdown with "Service" and "KSVC" options
 
 
+        @smoke 
+        Scenario: Add Trigger to Broker: KE-04-TC02
+            Given user is having Broker "broker-test1" on the Topology page
+              And user is at Topology page
```
Comment (sanketpathak) [inline frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-trigger.feature:24]: Mention subscriber in the given statement:
user has created knative service "nodejs-ex-git"

**Comment by sanketpathak** [comment]
Test runs are green
/lgtm
/approve

**Comment by SparshKesari** [comment]
Pr only affects E2E tests
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**